index.html

```diff
@@ -57,102 +55,126 @@ <h5>location</h5>
                     </div>
                     <div class="account-con">
                         <div class="user">
-                            <i class="fa-regular fa-user"></i>
+                            <a href="/screens/log-in.html"><i class="fa-regular fa-user text-md"></i></a>
                         </div>
                     </div>
+                    <div class="wish-con">
+                        <a href="/screens/wishlist-page.html">
+                            <span class="badge-icon badge-container"><i
+                                    class="fas fa-heart badge-icon-base text-md"></i>
+                                <span class="count-badge heart-badge-number">4</span>
+                            </span>
+                        </a>
+                    </div>
                     <div class="cart-con">
-                        <i class="fa-solid fa-cart-shopping thin"></i>
+                        <a href="/screens/cart-page.html">
+                            <span class="badge-icon badge-container"><i class="fa-solid fa-cart-shopping text-md"></i>
+                                <span class="count-badge cart-badge-number">3</span>
+                            </span>
+                        </a>
+                    </div>
+                </div>
+                <div class="bars-con">
+                    <div class="user">
+                        <a href="/screens/log-in.html"><i class="fas fa-bars"></i></a>
                     </div>
                 </div>
             </div>
+            <div class="search-btm p-1">
+                <form action="#" class="form-search flex-row-sb">
+                    <input type="text" class="text-rg pl-1" id="search-field" placeholder="search the entire store...">
+                    <button type="submit" class="btn-icon pr-sm"><i
+                            class="fa-solid fa-magnifying-glass search-icon thin"></i></button>
+                </form>
+            </div>
         </header>
 
         <main>
-            <div class="grid-6-col-auto pet-cards py-1 px-2">
+            <div class="grid-6-col-auto pet-cards py-1 px-2 hori-scroll">
                 <div class="p-card center">
                     <a href="#" class="p-img center">
-                        <img src="https://www.zigly.com/media/wysiwyg/for_dogs.png" alt="" class="img-responsive">
+                        <img src="https://www.zigly.com/media/wysiwyg/for_dogs.png" alt=""
+                            class="min-img-small img-responsive">
```

Comment:
Use the alt tag and give it text which relates to the image being shown, 
this will massively improve accessibility and screen reader support on your website.

Noticed it in many other places, hence will suggest to go through all HTML files again with a finecomb and resolve this.